spot-ml/src/main/scala/org/apache/spot/dns/model/DNSSuspiciousConnectsModel.scala

```diff
@@ -206,8 +207,21 @@ object DNSSuspiciousConnectsModel {
       config.localUser,
       config.analysis,
       config.nodes,
-      config.ldaPRGSeed)
-
+      config.ldaPRGSeed,
+      sparkContext,
+      sqlContext,
+      logger)
+
+    // Since DNS is still broadcasting ip to topic mix, we need to convert data frame to Map[String, Array[Double]]
```

Comment:
going forward, we should evaluate having all pipelines use the same structure.... presumably catalyst optimzer knows when to use broadcast joins and when to do other kinds of joins